bin/tinyreq

```diff
@@ -26,17 +27,20 @@ new clp({
   , examples: [
         "tinyreq -u http://ionicabizau.net"
     ]
-}, [urlOpt, methOpt, dataOpt, fieldsOpt]);
+}, [urlOpt, methOpt, dataOpt, fieldsOpt, quietOpt]);
 
 // Merge the options
 let options = ul.merge({
     method: methOpt.value || undefined
   , url: urlOpt.value || undefined
   , data: dataOpt.value || undefined
+  , quiet: quietOpt.value = quietOpt.is_provided
 }, {
     method: "GET"
 });
-
+if (options.quiet) {
+  logger.config.level = -1;
```

Comment:
@ArseniyAndru Whoups, sorry. I am wrong. [`-1` will stop everything](https://github.com/IonicaBizau/node-bug-killer/blob/b6e3e5e9731bb400bd1d67e080d2c2ca79c87b0b/lib/index.js#L89-L91). However, maybe we do want to show the errors.

In that case it should be `1` (double check that).